docs/animation/implementing.md

```diff
@@ -0,0 +1,42 @@
+Implementing the API
+======================
+
+Depending on what you want to animate with the API, code-side implementation is a bit different.
+Documentation on the ASM api itself (for controlling the animation) is found on the [ASM][asm] page because it is independent of what
+you are animation.
```

Comment:
Shouldn't it be animating?

Reply:
Please make a thread on http://www.minecraftforge.net/forum/forum/70-modder-support/

Reply:
???
I just wanted to help with spelling mistakes... 
I wasn't even asking questions...

Reply:
Im sorry I misinterpreted what you said and posted. Yes that should be "animating" instead of animation.

Reply:
No Problem...
I modified it in hopes of making it clearer to others, to ehat I'm referring to.

Reply:
Fixed.